geotrek/tourism/parsers.py

```diff
@@ -374,6 +374,12 @@ def filter_contact(self, src, val):
         ] if line]
         return '<br>'.join(lines)
 
+    def filter_capacity(self, src, val):
+        if val.isnumeric():
+            return int(val)
+        else:
+            return None
```

Comment:
Use `self.add_warning()` to output a warning stating that capacity received a bad value